pkg/scheduler/framework/unit_runtime/unit_framework.go

```diff
@@ -348,8 +348,9 @@ func (f *UnitFramework) scheduleOneUnitInstance(ctx context.Context, scheduledIn
 	runningUnitInfo *core.RunningUnitInfo, unitKey string, queuePriorityScore float64, unitCycleState, commonPreemptionState *framework.CycleState,
 	nodeGroup framework.NodeGroup,
 	usr *framework.UnitSchedulingRequest,
-) (bool, error) {
+) (success bool, err error) {
 	switchType, subCluster := f.handle.SwitchType(), f.handle.SubCluster()
+	godelScheduler := f.schedulerHooks.PodScheduler()
```

Comment:
Why is this initialization being advanced? If it is necessary to do, should we modify `preemptOneUnitInstance` as well?